build_mobilenet_base.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+
+if [ -z $1 ]; then
+    echo "Please specify wasmedge path"
+else
+    WASMEDGE=$1
+    WASI_NN_DIR=$(dirname "$0" | xargs dirname)
+    WASI_NN_DIR=$(realpath $WASI_NN_DIR)
+    WASMEDGE=$(realpath $WASMEDGE)
+    source /opt/intel/openvino_2021/bin/setupvars.sh
+
+    FIXTURE=https://github.com/intel/openvino-rs/raw/main/crates/openvino/tests/fixtures/mobilenet
+    pushd $WASI_NN_DIR/rust/
+    mkdir -p $WASI_NN_DIR/rust/examples/mobilenet-base/build
+    RUST_BUILD_DIR=$(realpath $WASI_NN_DIR/rust/examples/mobilenet-base/build/)
+    pushd examples/mobilenet-base
+    cargo build --release --target=wasm32-wasi
+    cp target/wasm32-wasi/release/mobilenet-base-example.wasm $RUST_BUILD_DIR
+    pushd build
+
+    if [ ! -f $RUST_BUILD_DIR/mobilenet.bin ]; then
```

Comment:
You download the assets here. Why did you commit the assets?

Reply:
Cause I think `openvino-rs` may modify the fixtures someday, and affect our demo's output.